nix/README.md

```diff
@@ -168,7 +168,7 @@ give you some more background and details on how it works.
 
 ### `default.nix`
 
-[`default.nix`](../default.nix) is our 'respository expression' that pulls all
+[`default.nix`](../default.nix) is our 'repository expression' that pulls all
```

Comment:
Just two random typos I ran across and didn't feel like opening up another PR for.